packages/node/src/ethereum/api.ethereum.ts

```diff
@@ -110,6 +110,7 @@ export class EthereumApi implements ApiWrapper {
     private endpoint: string,
     private blockConfirmations: number,
     private eventEmitter: EventEmitter2,
+    private unfinalizedBlocks: boolean,
```

Comment:
To minimise other changes we can default this to false.
```suggestion
    private unfinalizedBlocks = false,
```